docker/formbricks.sh

```
@@ -1,384 +1,57 @@
-  read -p "Are you sure you want to uninstall Formbricks? This will delete all the data associated with it! (yes/no): " uninstall_confirmation
-  uninstall_confirmation=$(echo "$uninstall_confirmation" | tr '[:upper:]' '[:lower:]')
-  if [[ $uninstall_confirmation == "yes" ]]; then
-    cd formbricks
-    sudo docker compose down
-    cd ..
-    sudo rm -rf formbricks
-    echo "🛑 Formbricks uninstalled successfully!"
-  else
-    echo "❌ Uninstalling Formbricks has been cancelled."
+  # Check if domain is valid
+  if ! [[ "$domain_name" =~ ^[a-zA-Z0-9.-]+$ ]]; then
```

Shellcheck

[warning] domain_name is referenced but not assigned.

(SC2154)